### PR TITLE
wu 3.10.0 (new)

### PR DESCRIPTION
**Pull request by chiefy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wu.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Wu < Formula
+  desc "cli that retrieves weather data from Weather Underground."
+  homepage "https://github.com/sramsay/wu"
+  url "https://github.com/sramsay/wu/archive/3.10.0.tar.gz"
+  sha256 "0934771cdd04cb4c38b45190dbfa97fa8b3aafaf53ea75152f070c97de850223"
+
+  depends_on "go"
```
Comment (bfontaine) [inline Formula/wu.rb]: This depends on Go for the build only, so please use `depends_on "go" => :build`.

File: Formula/wu.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "station": "Lincoln, NE",
+      "degrees": "F"
+    }
+    EOS
+  end
+
+  test do
+    ENV["GOPATH"] = testpath.realpath
+    assert_match "You must create a .condrc file in $HOME.", shell_output("#{bin}/wu -all")
+    system "false"
```
Comment (bfontaine) [inline Formula/wu.rb]: You can remove this line.

File: Formula/wu.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      "key": "YOUR_API_KEY",
+      "station": "Lincoln, NE",
+      "degrees": "F"
+    }
+    EOS
+  end
+
+  test do
+    ENV["GOPATH"] = testpath.realpath
```
Comment (bfontaine) [inline Formula/wu.rb]: You shouldn’t need this.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Review by bfontaine — APPROVED** [review]

File: Formula/wu.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+class Wu < Formula
+  desc "cli that retrieves weather data from Weather Underground."
```
Comment (MikeMcQuaid) [inline Formula/wu.rb]: `CLI`

File: Formula/wu.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    mkdir_p buildpath/"src/github.com/sramsay"
+    ln_sf buildpath, buildpath/"src/github.com/sramsay/wu"
+
+    cd "src/github.com/sramsay/wu" do
+      system "go", "build", "-o", bin/"wu", "-ldflags", "-X main.Version=#{version}"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    In order for `wu` to work, you need to obtain a Weather Underground API key:
```
Comment (MikeMcQuaid) [inline Formula/wu.rb]: I don't think we need these caveats as none of them are Homebrew-specific information.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/wu.rb (hunk 5 above)
Comment (chiefy) [inline Formula/wu.rb]: @MikeMcQuaid how else should I inform the installer that there are these requirements?

Comment (MikeMcQuaid) [inline Formula/wu.rb]: You should not. They can read the instructions prominently at the top of the homepage: https://github.com/sramsay/wu

Comment (chiefy) [inline Formula/wu.rb]: @MikeMcQuaid removed. Unless I missed it, might be cool to have something in the docs about what you should / shouldn't display to the user?

Comment (MikeMcQuaid) [inline Formula/wu.rb]: Where would be most useful, http://www.rubydoc.info/github/Homebrew/brew/master/Formula#caveats-instance_method?

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**